manifests/database/postgresql.pp

```diff
@@ -24,4 +24,11 @@
     encoding => 'utf8',
     locale   => 'en_US.utf8',
   }
+
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '7' {
+    file { '/etc/systemd/system/rh-postgresql12-postgresql.service.d':
```

Comment:
To take effect, this needs to trigger a `daemon-reload` once, but not sure how to do this from here?

Reply:
Notify the service?